make-up/code/5094-viktor-lozanovski/src/app/app-routing.module.ts

```diff
@@ -1,10 +1,15 @@
 import { NgModule } from '@angular/core';
 import { RouterModule, Routes } from '@angular/router';
 import { RecipeListComponent } from './recipe-list/recipe-list.component';
-
+import { RecipeDetailsComponent } from './recipe-details/recipe-details.component';
+import { RecipeAddComponent } from './recipe-add/recipe-add.component';
+import { RecipeEditComponent } from './recipe-edit/recipe-edit.component'; 
 const routes: Routes = [
+  { path: '', redirectTo: 'recipies', pathMatch: 'full'},
   { path: 'recipes', component: RecipeListComponent },
```

Comment:
+2 for the routes, only some of the are defined, and they don't work without a `router-outlet`